maven-core/src/main/java/org/apache/maven/graph/DefaultProjectDependencyGraph.java

```diff
@@ -158,4 +183,12 @@ public String toString()
         return sorter.getSortedProjects().toString();
     }
 
+    private class MavenProjectComparator implements Comparator<MavenProject>
+    {
+        @Override
+        public int compare( MavenProject o1, MavenProject o2 )
+        {
+            return order.get( o1 ) - order.get( o2 );
```

Comment:
This involves boxing, is is cheaper to get the `intValue()` from the `Integer` instance?

Reply:
I don't think there's any performance difference between unboxing and calling `intValue()`...